docs(self-hosted): describe production setup (#4645)
zitadel/zitadel: docs/docs/guides/deploy/overview.mdx

---

Choose your platform and run ZITADEL with the most minimal configuration possible.

- [Linux](./linux)
- [MacOS](./macos)
- [Docker Compose](./compose)
- [Knative](./knative)
- [Kubernetes](./kubernetes)

## Prerequisites

> - When run as a proof of concept, ZITADEL does not need many resources, 1 CPU and 512MB memory are more than enough. (With more CPU, the password hashing might be faster)

Reviewer: ```suggestion
- For test environments, ZITADEL does not need many resources, 1 CPU and 512MB memory are more than enough. (With more CPU, the password hashing might be faster)
```

- A cockroachDB or Postgresql (currently in beta) as only needed storage

## Releases

The easiest way to use ZITADEL is to run one of our container releases

- ZITADEL does provide latest and stable [container images](https://github.com/zitadel/zitadel/pkgs/container/zitadel)
- **stable** is the current **production** release of ZITADEL.

> - **latest** is the **last created** release from our pipelines that gets updated on a high frequency.

Reviewer: ```suggestion
- **latest** is the **last created** release from our pipelines that gets updated in a high frequency.
```

# Production Setup

> As soon as you successfully created your first proof of concept using one of the deployment guides in this section,

Reviewer: ```suggestion
As soon as you successfully created your first test environment using one of the deployment guides in this section,
```

you might want to configure ZITADEL for production and embed it into your system landscape.
To do so, jump straight to the [production setup guide](../manage/self-hosted/production).

> For achieving high availablility, we recommend to use [Kubernetes](https://kubernetes.io/docs/home/).
> We have an official [Helm chart](https://artifacthub.io/packages/helm/zitadel/zitadel) for easy deployment and maintenance.

Reviewer: ```suggestion
To achieving high availability, we recommend to use a [Kubernetes](https://kubernetes.io/docs/home/) Cluster.
We have an official [Helm chart](https://artifacthub.io/packages/helm/zitadel/zitadel) for easy deployment and maintenance.
```

Author: I like availabilility 😄
